src/features/caip/tokens.ts

```diff
@@ -100,11 +106,14 @@ export function resolveAssetNamespace(
   isSpl2022?: boolean,
 ) {
   if (isNative) return AssetNamespace.native;
-  if (protocol === ProtocolType.Ethereum) {
-    return isNft ? AssetNamespace.erc721 : AssetNamespace.erc20;
-  } else if (protocol === ProtocolType.Sealevel) {
-    return isSpl2022 ? AssetNamespace.spl2022 : AssetNamespace.spl;
-  } else {
-    throw new Error(`Unsupported protocol: ${protocol}`);
+  switch (protocol) {
+    case ProtocolType.Ethereum:
+      return isNft ? AssetNamespace.erc721 : AssetNamespace.erc20;
+    case ProtocolType.Sealevel:
+      return isSpl2022 ? AssetNamespace.spl2022 : AssetNamespace.spl;
+    case ProtocolType.Cosmos:
+      return AssetNamespace.ibcDenom;
```

Comment:
Will we only support ibc denom tokens for cosmos or is this temporary. If so please leave a `TODO cosmos` comment so I can keep track of what needs to change